src/app/components/Feedback/Feedback.jsx

```diff
@@ -28,12 +31,29 @@ class Feedback extends React.Component {
 
     return (
       <div className="feedback">
-        <button className="feedback-button" onClick={() => this.toggleForm()}>
+        <button
+          className="feedback-button"
+          onClick={() => this.openForm()}
+          aria-haspopup="true"
+          aria-expanded={showForm}
+          aria-controls="feedback-menu"
+        >
           Feedback
         </button>
-        <div className={`feedback-form-container${showForm ? ' active' : ''}`}>
+        <button
```

Comment:
Does this button also need `aria-haspopup`, `aria-expanded`, and `aria-controls`?

Reply:
I dont think the `<button>` would get `aria-expanded` the widget it is controlling would get that attribute. 
At least that's my reading of the [spec](https://www.w3.org/TR/wai-aria/states_and_properties#aria-expanded)

I think the `<button>` gets `aria-controls="theID"` ([here is that spec](https://www.w3.org/TR/wai-aria/states_and_properties#aria-controls))

Reply:
Some good examples here: http://heydonworks.com/practical_aria_examples/

`aria-haspopup` is what we want here. I don't think `aria-expanded` is needed in this scenario. We should maybe review what's in the toolkit

Reply:
From my understanding, the `button` would get `aria-expanded` and in this case, since both the open and the close buttons control the widget, they would both get it. I don't think `haspopup` is necessary for the close button, though.

Reply:
@seanredmond the example you shared uses `aria-expanded` instead of `aria-haspopup`. That is used for the dropdowns.